# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1488815904 18000
#      Mon Mar 06 10:58:24 2017 -0500
# Node ID 184ab9034ad53817a86688eda03a6a88a56e79a7
# Parent  aaf5ac82979d926f21ebff973c8240187912903c
* grabcode.m: Fix typo in doc string.

diff --git a/scripts/general/grabcode.m b/scripts/general/grabcode.m
--- a/scripts/general/grabcode.m
+++ b/scripts/general/grabcode.m
@@ -19,17 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn  {} {} grabcode (@var{url})
 ## @deftypefnx {} {} grabcode (@var{filename})
 ## @deftypefnx {} {@var{code_str} =} grabcode (@dots{})
 ##
 ## Grab the code from a report created by the @code{publish} function.
 ##
 ## The grabbed code inside the published report must be enclosed by the
-## strings @samp{##### SOURCE BEGIN #####} and @samp {##### SOURCE END #####}.
+## strings @samp{##### SOURCE BEGIN #####} and @samp{##### SOURCE END #####}.
 ## The @code{publish} function creates this format automatically.
 ##
 ## If no return value is requested the code is saved to a temporary file and
 ## opened in the default editor.  NOTE: The temporary file must be saved under
 ## a new or the code will be lost.
 ##
 ## If an output is requested the grabbed code will be returned as string
 ## @var{code_str}.
